src/sst/elements/vanadis/os/syscall/futex.cc

```diff
@@ -123,13 +123,14 @@ VanadisFutexSyscall::VanadisFutexSyscall( VanadisNodeOSComponent* os, SST::Link*
 
 void VanadisFutexSyscall::wakeWaiter(VanadisSyscallFutexEvent* event) const
 {
+    printf("Calling findFutex from wakeWaiter\n");
```

Comment:
Please grep for all of your printf additions and ensure that they aren't left over from debugging.